sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1210,6 +1210,23 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("SHOW TBLPROPERTIES") {
+    val t = "testcat.ns1.ns2.tbl"
+    withTable(t) {
+      spark.sql(s"CREATE TABLE $t (id bigint, data string) USING foo")
+
+      val e1 = intercept[AnalysisException] {
+        sql(s"SHOW TBLPROPERTIES $t")
+      }
+      assert(e1.message.contains("SHOW TBLPROPERTIES is only supported with v1 tables"))
```

Comment:
I think we can implement SHOW TBLPROPERTIES with v2 APIs. You can refer to https://github.com/apache/spark/pull/26166 about how to implement a v2 command.